src/main/java/am/aua/library/database/Database.java

```diff
@@ -96,9 +103,9 @@ public static void setDirectory(String directory) throws DatabaseException {
                 }
             }
 
+            createDatabase(DEFAULT_BOOKS_DATABASE, "[]");
```

Comment:
We are going to replace this part with a default static file which will come pre-shipped with the application